common/unsafe/src/test/scala/org/apache/spark/unsafe/types/CollationFactorySuite.scala

```diff
@@ -106,21 +106,27 @@ class CollationFactorySuite extends AnyFunSuite with Matchers { // scalastyle:ig
       ("UTF8_BINARY_AI","UTF8_BINARY"),
       ("UTF8_BINARY_CI_AI","UTF8_BINARY"),
       ("UTF8_BINARY_AI_CI","UTF8_BINARY"),
+      ("UTF8_BINARY_AI_RTRIM", "UTF8_BINARY_RTRIM"),
+      ("UTF8_BINARY_CI_RTRIM","UTF8_BINARY_RTRIM"),
+      ("UTF8_BINARY_AI_CI_RTRIM","UTF8_BINARY_RTRIM"),
```

Comment:
let's be consistent and use spaces:
```suggestion
      ("UTF8_BINARY_CI_RTRIM", "UTF8_BINARY_RTRIM"),
      ("UTF8_BINARY_AI_CI_RTRIM", "UTF8_BINARY_RTRIM"),
```